src/home/components/Form/index.tsx

```diff
@@ -25,10 +28,11 @@ const Form = ({ handleSubmit, loading }: FormProps): ReactElement => {
     <Box flexGrow={1}>
       <form onSubmit={handleFormSubmit}>
         <Flex flexDirection="column">
```

Comment:
Se não for usar nada mais do `Flex`, vale substituir pelo `Box`:
```suggestion
        <Box>
```

Reply:
Acho q de fato não vou mesmo, vou dar um change aqui.

Reply:
Aqui foi feito